### PR TITLE
PTV-458: Update app to use new refseq genome data

### PR DESCRIPTION
**Pull request by arfathpasha**
- The cog files were regenerated using the new refseq genome data from public workspace ReferenceDataManager in prod 
- The tests were updated to use the new refseq genome ids.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,6 +4,13 @@ MAINTAINER KBase Developer
 
 # Insert apt-get instructions here to install
 # any required dependencies for your module.
+RUN echo 1
```
Comment (sychan) [inline Dockerfile]: What is this for?

File: Dockerfile [hunk 2]
```diff
@@ -4,6 +4,13 @@ MAINTAINER KBase Developer
...
+
+# update jars
+RUN cd /kb/dev_container/modules/jars \
+	&& git pull \
+	&& . /kb/dev_container/user-env.sh \
+	&& make deploy
 
 # RUN apt-get update
```
Comment (sychan) [inline Dockerfile]: Seems to be no longer accurate

File: kbase.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ module-version:
     0.0.7
```
Comment (sychan) [inline kbase.yml]: Do we need to bump up the version?

**Comment by scanon** [comment]
This looks good to me.

**Pull request merged**